src/controllers/tripsController.js

```diff
@@ -49,7 +49,7 @@ export default class requestController {
             passportName,
             role
           }],
-          status: 'Open'
+          status: 'Open',
```

Comment:
Then this should change too

Reply:
let me change the code so that the data always inter the database as small letter even tho it wasn't entered as small letters

Reply:
I solved it, now when someone input data like a city, it will directly be converted to small letters